sessionctx/variable/sysvar.go

```diff
@@ -594,30 +590,19 @@ var defaultSysVars = []*SysVar{
 	{ScopeGlobal | ScopeSession, "min_examined_row_limit", "0"},
 	{ScopeGlobal, "sync_frm", "ON"},
 	{ScopeGlobal, "innodb_online_alter_log_max_size", "134217728"},
+	/* TiDB specific variables */
 	{ScopeSession, TiDBSnapshot, ""},
-	{ScopeGlobal | ScopeSession, DistSQLScanConcurrencyVar, "10"},
-	{ScopeGlobal | ScopeSession, DistSQLJoinConcurrencyVar, "5"},
 	{ScopeSession, TiDBSkipConstraintCheck, "0"},
-	{ScopeSession, TiDBSkipDDLWait, "0"},
-	{ScopeGlobal | ScopeSession, TiDBSkipUTF8Check, "0"},
-	{ScopeSession, TiDBOptAggPushDown, "ON"},
-	{ScopeSession, TiDBOptInSubqUnFolding, "OFF"},
-	{ScopeSession, BuildStatsConcurrencyVar, "4"},
+	{ScopeSession, TiDBOptAggPushDown, boolToIntStr(DefOptAggPushDown)},
+	{ScopeSession, TiDBOptInSubqUnFolding, boolToIntStr(DefOptInSubqUnfolding)},
+	{ScopeSession, TiDBBuildStatsConcurrency, strconv.Itoa(DefBuildStatsConcurrency)},
+	{ScopeGlobal | ScopeSession, TiDBDistSQLScanConcurrency, strconv.Itoa(DefDistSQLScanConcurrency)},
+	{ScopeGlobal | ScopeSession, TiDBIndexLookupSize, strconv.Itoa(DefIndexLookupSize)},
+	{ScopeGlobal | ScopeSession, TiDBIndexLookupConcurrency, strconv.Itoa(DefIndexLookupConcurrency)},
+	{ScopeGlobal | ScopeSession, TiDBSkipDDLWait, boolToIntStr(DefSkipDDLWait)},
```

Comment:
TiDBSkipDDLWait should be session scope only. It is not safe to make it global scope.